core/src/main/java/org/bitcoinj/base/VarInt.java

```diff
@@ -63,20 +63,19 @@ public static VarInt ofBytes(byte[] buf, int offset) throws ArrayIndexOutOfBound
     public static VarInt read(ByteBuffer buf) throws BufferUnderflowException {
         buf.order(ByteOrder.LITTLE_ENDIAN);
         int first = Byte.toUnsignedInt(buf.get());
+        int originallyEncodedSize = Byte.BYTES;
```

Comment:
To avoid mutating `originallyEncodedSize`, I would remove this line and add a `Byte.BYTES + ` to the three lines inside the conditionals.

Reply:
Actually, the constants are used again down in `sizeOf()`. Maybe we should just define them as private constants in the file, e.g.
```
private static int VARINT_BYTE_SIZE = Byte.BYTES;
private static int VARINT_SHORT_SIZE = Byte.BYTES + SHORT.BYTES;
```
etc.